.my_data/data_nodes/DATANODE_input_995718a9-f47a-4160-881c-fe8107fed7ab.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This should not be committed. The `.my_data` folder should have been removed by the test, and maybe we can git ignore `.my_data/` folder